.github/workflows/ci.yml

```diff
@@ -0,0 +1,221 @@
+---
+name: "CI"
+on:
+  - "push"
+  - "pull_request"
+
+jobs:
+  black:
+    runs-on: "ubuntu-20.04"
+    env:
+      INVOKE_NAUTOBOT_CHATOPS_LOCAL: "True"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v2"
+      - name: "Set up Python"
+        uses: "actions/setup-python@v2"
+        with:
+          python-version: "3.9"
+      - name: "Install Python Packages for Setup"
+        run: "pip install invoke poetry"
+      - name: "Install Python Packages for Linting"
+        run: "poetry install"
```

Comment:
As commented, it would be good to optimize it and do not install the full package, only the `invoke` and `black`.

Reply:
``` 
      - name: "Install Python Packages for Linting"
        run: "pip install --upgrade pip wheel invoke toml flake8"
      - name: "Linting: flake8"
        run: "invoke flake8"
```

Reply:
That mechanism as shown is not going to work @chadell , that will install the latest vs what is defined in the dev install. I think we have a working pattern though.